### ISSUE TITLE
Big Sur 11.4 YouTube and Netfilx not working

### ISSUE THREAD

**Issue by scylscyl** [icaros7/OpenCore_Surface_Pro_4#7]
Netfilx on Safari keeps giving an error.
When I open a video on YouTube on Safari, the PC crashes. I have to restart.

**Comment by icaros7** [comment]
I'm so sorry for late reply.

I found the same thing. This is expected by hardware acceleration on safari.
Because when I try to watch video what has some format, It's normally.

I will check my system log.

**Comment by Giant-Li** [comment]
Hello icaros7, did you have any progress on it? I think I have the same problem. I can open the YouTube website, but the video stucks on loading once I open a specific video. And the whole system becomes slow even I close safari until I reboot the system.

**Comment by icaros7** [comment]
Hello, Long time no see.

I want to watch Youtube or Netflix on my SP4, but now I recognized what cause this problem.

https://dortania.github.io/OpenCore-Post-Install/universal/drm.html
```
DRM is broken for iGPU-only systems
 - This could be fixed with Shiki (now WhateverGreen) til 10.12.2, but broke with 10.12.3
 - This is due to the issue that **our iGPUs don't support Apple's firmware and that our Management Engine (opens new window) doesn't have Apple's certificate**
```

What as far as I understand, we can't fix this problem without eGPU or downgrade to ~10.12.2.

Thank you :)
